packages/sdk/src/TokenboundClient.ts

```diff
@@ -539,76 +457,75 @@ class TokenboundClient {
    * @returns a Promise that resolves to the transaction hash of the executed call
    */
   public async transferERC20(params: ERC20TransferParams): Promise<`0x${string}`> {
-    const { 
+    const {
       account: tbAccountAddress,
       amount,
       recipientAddress,
       erc20tokenAddress,
       erc20tokenDecimals,
     } = params
 
-    if(erc20tokenDecimals < 0 || erc20tokenDecimals > 18) throw new Error("Decimal value out of range. Should be between 0 and 18.")
+    if (erc20tokenDecimals < 0 || erc20tokenDecimals > 18)
+      throw new Error('Decimal value out of range. Should be between 0 and 18.')
 
     const amountBaseUnit = parseUnits(`${amount}`, erc20tokenDecimals)
 
-    const recipient = recipientAddress.endsWith(".eth")
-      ? await this.publicClient.getEnsResolver({name: normalize(recipientAddress)})
-      : recipientAddress
+    // const recipient = recipientAddress.endsWith('.eth')
+    //   ? await this.publicClient.getEnsResolver({ name: normalize(recipientAddress) })
+    //   : recipientAddress
```

Comment:
Looks like we were perviously allowing for ENS and resolve it to an address. Are we not allowing that anymore within the SDK?